foundry.toml

```diff
@@ -6,8 +6,20 @@ optimizer = true
 optimizer_runs = 200
 libs = ['lib']
 src = 'src'
-test = 'test'
+test = 'tests'
 out = 'out'
+bytecode_hash = 'none'
```

Comment:
You could remove this. It was very specific to Vertex

Reply:
Yeah, I thought about it but at the same time maybe there's conflicts with address generation in the future, so kinda torn hah.